src/Core/Geometry/TopologicalMesh.cpp

```diff
@@ -463,8 +466,11 @@ void TopologicalMesh::initWithWedge( const TriangleMesh& triMesh ) {
                              m_wedges.m_wedgeVector3AttribHandles,
                              m_wedges.m_wedgeVector4AttribHandles,
                              &wd );
-
-        m_wedges.m_data.emplace_back( wd );
+        // here ref is not incremented
+        w.setWedgeData( std::move( wd ) );
+        // the newly added wedge is not referenced yet, will be done with `newReference` when
+        // creating faces just below
+        m_wedges.m_data.emplace_back( w );
```

Comment:
Is it push or emplace here ?

Reply:
right, it was emplace, but with the fix, it's push. Fixed.